CHANGES.txt

```diff
@@ -1,12 +1,15 @@
 V3.7.0:
   Users:
      - Allowing drag and drop a coordinate when picking
+     - Allowing to create subsets from the selection in the metadataviewer plotter
+     - Fixing an error creating subset through new metadataviewer
+
+Developers:
```

Comment:
This is misplaced yun! Developers should go before the "Plugin._defineVar...." line